### PR TITLE
Fom MANE overview, link out to gene overview

### LINKED ISSUE #367: Link out to gene overview from MANE genes overview
Add a link on the gene name so when you click you can open the gene overview page - on another tab

### PR DESCRIPTION
**Pull request by northwestwitch**
### Added/Changed/Fixed
- Adds links to gene overview in the MANE overview - Closes #367 

image

### How to test
- [x] Deploy on stage server and make sure gene links on MANE page work

## Review
- [x] Tests executed by CR
- [ ] "Merge and deploy" approved by

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
mmm in this PR looks like I broke the first form, where you customise the search.

File: src/chanjo2/templates/mane-overview.html [hunk 1]
```diff
@@ -27,6 +27,8 @@ <h2 class="accordion-header" id="flush-headingOne">
 			<!-- hidden fields passed from previous query -->
 			<form name="customizeForm" action="{{url_for('mane_overview')}}" method="post">
 			<input type="hidden" name="build" value="GRCh38"/>
+			<input type="hidden" name="default_level" id="defaultLevel" value="{{extras.default_level}}"/>
```
Comment (northwestwitch) [inline src/chanjo2/templates/mane-overview.html:30]: Default level is not used in the MANE overview report, but in the gene overview, and this is a way to pass along the value

**Comment by Jakob37** [comment]
> I've got one small PR that adds gene links to the MANE report. It should work but you could give it a try if you want @Jakob37 . As a final step for this release I was thinking to address #366.

I'll try it out!

**Comment by Jakob37** [comment]
Looks like it is working!

pr_1

pr_2

On the gene page. Would it be possible to show what transcript is MANE as well? As the ID type differs, the users will not immediately see which one was "NM_001101.5" in the example above.

**Comment by northwestwitch** [comment]
> On the gene page. Would it be possible to show what transcript is MANE as well? As the ID type differs, the users will not immediately see which one was "NM_001101.5" in the example above.

Yes, I'll add it, but on my new PR, the one adding also exons. It will be easier

**Comment by northwestwitch** [comment]
Merging this one. Thanks @Jakob37!

**Pull request merged**